src/components/file-dropzone/file-dropzone.tsx

```diff
@@ -0,0 +1,162 @@
+import {
+    Component,
+    h,
+    Event,
+    EventEmitter,
+    Host,
+    Prop,
+    State,
+} from '@stencil/core';
+import { FileInfo } from '../../global/shared-types/file.types';
+import { createFileInfo, isTypeAccepted } from '../../util/files';
+
+/**
+ * This component enables you to seamlessly convert any region of the user interface into
+ * a file dropzone area, just by wrapping it inside the `limel-file-dropzone`.
+ *
+ * The file dropzone can then be used to allow end-users to upload files
+ * by dragging and dropping them into the specified area.
+ *
+ * After receiving the files, the component emits a `filesSelecgted` event. For unsupported
```

Comment:
```suggestion
 * After receiving the files, the component emits a `filesSelected` event. For unsupported
```

Reply:
I highly recommend everyone to install `streetsidesoftware.code-spell-checker`: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker